rest/route/github.go

```diff
@@ -263,11 +264,107 @@ func (gh *githubHookApi) Run(ctx context.Context) gimlet.Responder {
 		if event.GetAction() == githubActionChecksRequested {
 			return gh.handleMergeGroupChecksRequested(event)
 		}
+
+	case *github.CheckRunEvent:
+		if event.GetAction() == githubActionRerequested {
+			return gh.handleCheckRunRerequested(ctx, event)
+		}
 	}
 
 	return gimlet.NewJSONResponse(struct{}{})
 }
 
+func (gh *githubHookApi) handleCheckRunRerequested(ctx context.Context, event *github.CheckRunEvent) gimlet.Responder {
+	owner := event.Repo.Owner.GetLogin()
+	repo := event.Repo.GetName()
```

Comment:
should we check to make sure they are not `""`?

Reply:
I think it should be fine because we don't do that anywhere else in this file and we can assume that we always have owner repo from github events